src/Aspire.ProjectTemplates/templates/aspire-starter/.template.config/template.json

```diff
@@ -254,6 +254,48 @@
         }
       ]
     },
+    "XUnitVersion": {
+      "type": "parameter",
+      "description": "The version of xUnit.net to use.",
+      "displayName": "xUnit.net version",
+      "datatype": "choice",
+      "isEnabled": "(TestFramework == xUnit.net)",
```

Comment:
'xunit.net' causes VS to crash. So template engine does not want quotes here, but does require them in the csproj.

Reply:
@phenning thoughts?

Reply:
@danmoseley  to be clear the templates are working now right? This option is only visible in VS when xUnit.net is the selected testing framework?

Reply:
Can you attach a debugger and get a stack to see where the crash is coming from, or see if there is anything in the activity log?

Reply:
I opened issue linked below with stack. Incidentally the exception message (which isn't shown to the user) has a typo in it. I think it was something like "invalid chrcter" or similar.

LMK if you need more.

Reply:
@DamianEdwards yes all this works, since I removed those single quotes. This should be good to merge.

Reply:
No, it doesn't - sorry. @phenning would you be able to tell me why the condition on this line (in its non crashing form) does not work?

Reply:
OK, we tracked this down to a bug in the VS template engine- either it didn't work, or it crashes--
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/2417642

To work around it, I've renamed some identifiers. Identifiers in some cases can't be substrings of other identifiers, until this is fixed.. @DamianEdwards fyi.